appdev/guides/pushnotifications.rst

```diff
@@ -0,0 +1,235 @@
+Push notifications
+==================
+
+Let's assume that you have an app created with Clickable and published on the OpenStore. But now you want to be able to send Push Notifications to your users. First of all, you need to understand how this is working:
+
+.. image:: ../../_static/images/appdev/guides/pushnotifications/Diagram.png
+
+1. Every app which uses push notifications has got a unique token. This token identifies the user, the device and the app itself. The token is generated by the UBports Push Service.
+2. You will need the token to send a push notification so the app needs to send this token to the entity, which will send the notification.
```

Comment:
Would this be clearer as "the app sends its token to the app developer's server"?

Reply:
You're right. I have changed it